### ISSUE TITLE
bigger batchsize leads to unstable loss during training with transformer big model

### ISSUE THREAD

**Issue by zherowolf** [pytorch/fairseq#563]
Hi,guys.
I'm confused with the problem that when I use bigger batchsize (set update_freq bigger or use multi-nodes) , the loss will  be unstable within the first thousands steps. Have you ever had this problem?
Specifically，when I use 1 nodes with update_freq=8, the training is normal (bsz≈18k). When I use 2 nodes with update_freq=8 (now the bsz should be doubled, 35k)， the loss will be unstable. When the bsz ≈ 70k， the training is more unstable. that seems weird. Other hyper parameters are same.

as the following picture shows, the loss will increase somehow ,and decrease later.
![image](https://user-images.githubusercontent.com/41291189/54006465-6c573600-4198-11e9-9ac0-3f2dd2024432.png)

**Comment by huihuifan** [comment]
When you scale the batchsize, do you change the learning rate?

**Comment by zherowolf** [comment]
@huihuifan nope and that confused me. With same learning rate , bigger batchsize training is supposed to be more stable.

**Comment by jiezhangGt** [comment]
It is may be fp16's’ fault

**Comment by myleott** [comment]
Closing for now, please open a new issue following one of the new Issue templates if needed, thanks!
